src/carma-platform/parse_ros_bags.py

```diff
@@ -241,7 +245,11 @@ def get_object_with_id(id_, msg):
             ):
                 continue
 
-            cdasim_time_ms = math.floor((message.header.stamp - time_offset).to_sec() * 1_000)
+            try:
+                cdasim_time_ms = round((message.header.stamp - time_offset).to_sec() * 1_000)
+            except (TypeError):
+                cdasim_time_ms = 0
```

Comment:
Should we skip this iteration instead? If the time difference is negative, wouldn't that mean CDASim is not fully set up?